src/tilesource.js

```diff
@@ -275,28 +276,39 @@ $.TileSource.prototype = {
         return this._tileHeight;
     },
 
+    /**
+     * @function
+     * @param {Number} level
+     * @return {OpenSeadragon.TileSource} Chainable.
```

Comment:
We're actually moving away from making our functions chainable. Let's just leave this not returning anything.